verifiers/utils/dsse.go

```diff
@@ -19,6 +42,30 @@ func PayloadFromEnvelope(env *dsselib.Envelope) ([]byte, error) {
 	return payload, nil
 }
 
+func StatementFromPayload(payload []byte) (*intoto.Statement, error) {
+	var statement intoto.Statement
+	if err := json.Unmarshal(payload, &statement); err != nil {
+		return nil, fmt.Errorf("%w: %s", serrors.ErrorInvalidDssePayload, err.Error())
```

Comment:
nit: wrap both errors
```suggestion
		return nil, fmt.Errorf("%w: %w", serrors.ErrorInvalidDssePayload, err)
```